### ISSUE TITLE
[Admin dashboard] "Store" link points to the homepage instead of shopfront

### ISSUE THREAD

**Issue by andrewpbrett** [openfoodfoundation/wishlist#279]

**Comment by audez** [comment]
Hi! We have a user who contacted us because she doesn't understand why she can't see her shop when clicking on the "Store" button. @RachL do you think the design and dev of this will take time? if yes, would it be possible to remove the "Store" button meanwhile?

**Comment by lin-d-hop** [comment]
This could be a papercut... a bit of a stretch as we need the dropdown design for a multishop user. But maybe we could put this in a little background pile for Jess?

**Comment by RachL** [comment]
@lin-d-hop yes that's the plan. This is why it's not in the candidates column for now.

@audez I don't know how long it would take. Meanwhile I don't think it is a good move to remove it based on 1 user feedback. Some other users might still find it useful. We could plan an analytic event to dig deeper into it, but I wonder if it's worth putting so much effort in something that will change during the year?

**Comment by audez** [comment]
@RachL all right i understand, I thought removing could be quick and easy. I can't see how it can be useful to anyone because of the bug: this button was supposed to show the producer how the buyer will view the shop. But instead of landing on the shopfront, you land on OFN homepage. The issue was written in 2020 and I edited it in July based on other users feedback..

**Comment by RachL** [comment]
It's not a bug @audez The issue was written in 2020 but this button was here long before. Maybe it's a spree leftover (spree was about handling only 1 shop, it's OFN that handles several shops), but it was never designed to show the shopfront of an enterprise, only to go back to the homepage. In that case the button is working fine. Perhaps we can rename it "Home" / "Accueil" if it really introduce confusion?
